Cannot redeclare block-scoped variable '__DEBUG_BUILD__'

### **Environment**
How do you use Sentry?
Sentry SaaS (sentry.io)

Which SDK and version?

```
@sentry/angular: ^7.11.0
@sentry/capacitor: ^0.9.0

Ionic:

   Ionic Framework               : @ionic/angular 5.9.1
   @angular-devkit/build-angular : 0.1102.6
   @angular-devkit/schematics    : 11.2.7
   @angular/cli                  : 11.2.6
   @ionic/angular-toolkit        : 2.3.3

Capacitor:

   Capacitor CLI      : 3.3.2
   @capacitor/android : 3.3.2
   @capacitor/core    : 3.3.2
   @capacitor/ios     : 3.3.2
```


### **Steps to Reproduce**
We've never used @sentry/capacitor and always used @sentry/angular for web only. I tried to implement @sentry/capacitor after having upgraded to Capacitor 3. However, when trying to build the mobile apps (Android/iOS) I get the following errors:

```
Error: node_modules/@sentry/capacitor/node_modules/@sentry/types/types/globals.d.ts:2:11 - error TS2451: Cannot redeclare block-scoped variable '__DEBUG_BUILD__'.
const __DEBUG_BUILD__: boolean;
node_modules/@sentry/types/types/globals.d.ts:2:11
const __DEBUG_BUILD__: boolean;
'__DEBUG_BUILD__' was also declared here.

Error: node_modules/@sentry/types/types/globals.d.ts:2:11 - error TS2451: Cannot redeclare block-scoped variable '__DEBUG_BUILD__'.
const __DEBUG_BUILD__: boolean;
node_modules/@sentry/capacitor/node_modules/@sentry/types/types/globals.d.ts:2:11
const __DEBUG_BUILD__: boolean;
'__DEBUG_BUILD__' was also declared here.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,12 @@
 
 - Add support for Capacitor 4.0.0 ([#201](https://github.com/getsentry/sentry-capacitor/pull/201))
 
+### Fixes
+
+- build(javascript): Bump sentry-javascript, sentry-vue, sentry-react and sentry-angular dependencies to `7.12.1`. ([#???](https://github.com/getsentry/sentry-capacitor/pull/???))
+  - [changelog](https://github.com/getsentry/sentry-javascript/releases/tag/7.12.1)
+  - [diff](https://github.com/getsentry/sentry-javascript/compare/7.8.1...7.12.1)
+
 ## 0.9.0
 
 ### Fixes

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,7 +8,7 @@
 
 ### Fixes
 
-- build(javascript): Bump sentry-javascript, sentry-vue, sentry-react and sentry-angular dependencies to `7.12.1`. ([#???](https://github.com/getsentry/sentry-capacitor/pull/???))
+- build(javascript): Bump sentry-javascript, sentry-vue, sentry-react and sentry-angular dependencies to `7.12.1`. ([#214](https://github.com/getsentry/sentry-capacitor/pull/214s))
   - [changelog](https://github.com/getsentry/sentry-javascript/releases/tag/7.12.1)
   - [diff](https://github.com/getsentry/sentry-javascript/compare/7.8.1...7.12.1)
 

--- a/example/ionic-angular/package.json
+++ b/example/ionic-angular/package.json
@@ -14,7 +14,7 @@
     "@ionic-native/splash-screen": "^5.0.0",
     "@ionic-native/status-bar": "^5.0.0",
     "@ionic/angular": "^5.0.0",
-    "@sentry/angular": "^7.8.1",
+    "@sentry/angular": "^7.12.1",
     "@sentry/capacitor": "file:.yalc/@sentry/capacitor",
     "rxjs": "~6.5.5",
     "tslib": "^2.0.0",

--- a/package.json
+++ b/package.json
@@ -37,9 +37,9 @@
   "license": "MIT",
   "peerDependencies": {
     "@capacitor/core": ">=3.0.0",
-    "@sentry/angular": "7.8.1",
-    "@sentry/react": "7.8.1",
-    "@sentry/vue": "7.8.1"
+    "@sentry/angular": "7.12.1",
+    "@sentry/react": "7.12.1",
+    "@sentry/vue": "7.12.1"
   },
   "peerDependenciesMeta": {
     "@sentry/angular": {
@@ -53,13 +53,13 @@
     }
   },
   "dependencies": {
-    "@sentry/browser": "7.8.1",
-    "@sentry/core": "7.8.1",
-    "@sentry/hub": "7.8.1",
-    "@sentry/integrations": "7.8.1",
-    "@sentry/tracing": "7.8.1",
-    "@sentry/types": "7.8.1",
-    "@sentry/utils": "7.8.1",
+    "@sentry/browser": "7.12.1",
+    "@sentry/core": "7.12.1",
+    "@sentry/hub": "7.12.1",
+    "@sentry/integrations": "7.12.1",
+    "@sentry/tracing": "7.12.1",
+    "@sentry/types": "7.12.1",
+    "@sentry/utils": "7.12.1",
     "@sentry/wizard": "^1.1.4",
     "promise": "^8.1.0"
   },
@@ -68,8 +68,8 @@
     "@capacitor/core": "^3.0.1 || ^4.0.0",
     "@capacitor/ios": "^3.0.1 || ^4.0.0",
     "@ionic/prettier-config": "^1.0.0",
-    "@sentry-internal/eslint-config-sdk": "7.8.1",
-    "@sentry-internal/eslint-plugin-sdk": "7.8.1",
+    "@sentry-internal/eslint-config-sdk": "7.12.1",
+    "@sentry-internal/eslint-plugin-sdk": "7.12.1",
     "@sentry/typescript": "5.20.1",
     "@types/jest": "^26.0.15",
     "eslint": "^7.13.0",

